massdriver-application/main.tf

```diff
@@ -17,36 +17,35 @@ locals {
     secrets     = local.secrets
   })
 
-  application_identity_id = var.create_application_identity ? mdxc_application_identity.main.0.id : var.application_identity_id
+  application_identity_id = mdxc_application_identity.main.id
 
   policies = { for p in local.app_policies_queries : p => jsondecode(data.jq_query.policies[p].result) }
 
   # env vars that have been resolved. additional vars may be added per cloud
   base_envs = { for k, v in local.app_envs_queries : k => jsondecode(data.jq_query.envs[k].result) }
-    
+
   # Auto generate ENV Vars for each secret
   envs_with_secrets = merge(local.base_envs, local.secrets)
-    
-  # Needed by Azure k8s until Workload Identity is out of preview.
-  azure_envs = (local.is_azure_k8s && mdxc_application_identity.main[0] != null) ? mdxc_application_identity.main[0].azure_application_identity : {}
-  azure_aks_envs = local.is_azure_k8s ? {
-    AZURE_TENANT_ID : local.azure_envs.application_id
-    AZURE_CLIENT_ID : local.azure_envs.service_principal_client_id
-    AZURE_CLIENT_SECRET : local.azure_envs.service_principal_secret
-  } : {}
+
+  # App Serivce / Function App will auto-inject the secret, but still need these env-vars added.
```

Comment:
This is unfortunate. I was hoping all this ENV stuff could be removed instead of just shifting it from "needed by AKS" to "needed by App Service/Function App"

Reply:
Yeah totally!

Before it was needed by _all_.